### ISSUE TITLE
[Research] Conduct internal a11y audit for Directory

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-book-directory-fe#106]
Assess compliance with WCAG 2.1 AA guidelines for current staging/production directory using Axe browser tools and file tickets for any needed improvements. See brief demo 

https://user-images.githubusercontent.com/13485451/119711298-399ac600-be14-11eb-8c4a-e39de9e9061a.mp4

Document and identify any a11y failures and create tickets/issues for fixing them.

@greatislander showed some pattern libraries for an open source project: https://github.com/platform-coop-toolkit/ + https://github.com/platform-coop-toolkit/coop-library + https://github.com/platform-coop-toolkit/pinecone + https://pinecone.platform.coop/ 

He recommended looking at https://baseline.10up.com/components for front end design components + their guide https://10up.github.io/Engineering-Best-Practices/markup/#accessibility
